src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -499,7 +525,19 @@ void Instance::OnResult(Status commissioningError, CharSpan debugText, int32_t i
     memcpy(mLastNetworkID, mConnectingNetworkID, mLastNetworkIDLen);
     mLastNetworkingStatusValue.SetNonNull(commissioningError);
 
-    commandHandle->AddResponse(mPath, response);
+#if CONFIG_NETWORK_LAYER_BLE
+    bool supportsConcurrentConnection = CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION;
+    if (!supportsConcurrentConnection)
+    {
+        ChipLogProgress(NetworkProvisioning, "Instance::OnResult Non-Concurrent Mode, ConnectNetworkResponse will NOT be sent");    
```

Comment:
This is going to log errors in CommandHandler, as it tries to send a response and discovers there isn't a response to send. We should probably notify it, in a followup, to not expect any responses...

Reply:
Currently you see this, is it acceptable for the moment?

```
[1699270892.045538][1147:1147] CHIP:NP: Instance::OnResult Non-Concurrent Mode, ConnectNetworkResponse will NOT be sent
[1699270892.045602][1147:1147] CHIP:DMG: Endpoint 0, Cluster 0x0000_0030 update version to 869e5227
[1699270892.045639][1147:1147] CHIP:DMG: Decreasing reference count for CommandHandler, remaining 0
[1699270892.045753][1147:1147] CHIP:DMG: Failed to send command response: src/app/CommandHandler.cpp:221: CHIP Error 0x00000003: Incorrect state
[1699270892.045772][1147:1147] CHIP:DMG: Command handler moving to [AwaitingDe]
```

Reply:
It's ok as a temporary state, but not OK to ship.

Reply:
Ok to do this in a different ticket?

Reply:
Yes, that's why I said "in a followup" above.  Make sure the issue is filed and there are code comments pointing to it, please.

Reply:
Add CommandHandler issue reference #30576